app/models/form_profiles/va_0873.rb

```diff
@@ -42,11 +42,19 @@ def initialize_personal_information
     payload = user.full_name_normalized.merge(preferred_name:, service_number:)
 
     VA0873::FormPersonalInformation.new(payload)
+  rescue => e
+    log_exception_to_sentry(e, {}, prefill: :personal_information)
+    {}
   end
 
   def initialize_ava_profile
-    school = GIDSRedis.new.get_institution_details_v0(id: profile.school_facility_code)
-    school_name = school[:data][:attributes][:name]
+    school_name = nil
+
+    unless profile.school_facility_code.nil?
+      school = GIDSRedis.new.get_institution_details_v0(id: profile.school_facility_code)
+      school_name = school[:data][:attributes][:name]
+    end
```

Comment:
Can you add a test to `form_profile_spec`? There is a test there now that uses this value: `'schoolFacilityCode' => '12345678'`, so you could do one with `nil` as well.

Reply:
Updated.